assets/javascripts/src/components/pages/Contribution.jsx

```diff
@@ -90,7 +90,7 @@ class FormWithRecurring extends React.Component {
                 : 'One-off amount';
 
 
-        return <div className={'contribute-controls contribute-fields contribute-controls--recurring ' + (this.props.amounts.length % 3 ? 'option-button__fours ' : 'option-button__three ')}>
+        return <div className="contribute-controls contribute-fields contribute-controls--recurring option-button__fours">
```

Comment:
Fair enough - it's probably more to do with how I split out these components for A/B tests. I'm still trying to figure out the best way of doing it.